pandas/core/indexes/timedeltas.py

```diff
@@ -487,7 +488,8 @@ def get_loc(self, key, method=None, tolerance=None):
         -------
         loc : int
         """
-        if is_list_like(key):
+        if is_list_like(key) or (isinstance(key, datetime) and key is not NaT):
```

Comment:
not sure if we use isna else for NaT checking?

Reply:
I think in this context `isna` is less clear.  Since there is a specific na-like object we are catching here, we should be explicit about it.

Reply:
ok this comment looks good